gcsio/pom.xml

```diff
@@ -146,7 +150,23 @@
       <groupId>com.google.code.gson</groupId>
       <artifactId>gson</artifactId>
     </dependency>
-
+    <dependency>
+      <groupId>io.opencensus</groupId>
+      <artifactId>opencensus-impl</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.opencensus</groupId>
+      <artifactId>opencensus-exporter-stats-stackdriver</artifactId>
```

Comment:
Why we do not use https://search.maven.org/search?q=g:com.google.cloud.opentelemetry instead?

https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/blob/main/exporters/metrics/README.md

Reply:
I think that it's fine to use alpha library, this is just metrics, and we use alpha GCS gRPC client library anyway - if we know that this is a long term solution, I think that better to use it from get go, instead of migrating to it in the future - it will save us some work. Also, this library may not have dependency on Apache HTTP client that we want to remove from GCS connector 3.0 release

Reply:
On second look, grpc-java does support Opencensus metrics out of the box. I couldn't find opentelemetry integration yet. 

I think the migration would be worth the additional client metrics support as all the changes could be done behind metricsRecorder